pkg/util/util_test.go

```diff
@@ -0,0 +1,17 @@
+package util
+
+import "testing"
+
+func TestNamespaceOpenShiftObject(t *testing.T) {
+
+	name, err := NamespaceOpenShiftObject("bar", "foo")
```

Comment:
There should be more test cases. For example, tests where the component name and/or the application name is empty.